README.md

```diff
@@ -63,6 +63,15 @@ This file resolver caches the network files, so you will want to clear out the o
 Using a script like this would be a good idea: https://github.com/pulibrary/loris/blob/607567b921404a15a2111fbd7123604f4fdec087/bin/loris-cache_clean.sh
 By default the cache is located in `tmp/network_files`. You can set the cache path like this: `Riiif::Image.file_resolver.cache_path = '/var/cache'`
 
+### GraphicsMagick
+
+To use [GraphicsMagick](http://www.graphicsmagick.org/) instead of ImageMagick
+
+    Riiif::ImagemagickCommandFactory.external_command = "gm convert"
+    Riiif::ImageMagickInfoExtractor.external_command  = "gm convert"
```

Comment:
It is API compatible with IM, or intended to be, at least at the point it forked from IM, and I've tested that it is for riiif-issued commands.